apps/dh/api-dh/source/DataHub.WebApi/Clients/Wholesale/MeasurementsReports/MeasurementsReportsClient.cs

```diff
@@ -44,4 +60,12 @@ public async Task<IEnumerable<RequestedMeasurementsReportDto>> GetAsync(Cancella
 
         return responseApiContent.OrderByDescending(x => x.CreatedDateTime);
     }
+
+    private static bool IsPeriodAcrossMonths(MeasurementsReportRequestFilterDto measurementsReportRequestFilter)
+    {
+        var startDate = TimeZoneInfo.ConvertTimeBySystemTimeZoneId(measurementsReportRequestFilter.PeriodStart, "Romance Standard Time");
```

Comment:
Suggestion: I think nodetime can help here. So we dont need timezone

Reply:
I think it is hard to avoid the use of timezone here TBH 🤔 In the next gen of range date pickers, I think we should seriously consider dropping the time part and let the BFF convert it to the proper zone. Then we can have DateInterval and LocalDate to work with in the BFF, which is much nicer for these sort of things. But that is a discussion for another time 😄

Reply:
If we decide to change it, then we can look at both this one and Settlement reports because the same method is also used there. Can that be a separate PR?